builtin/commit.c

```diff
@@ -55,20 +55,19 @@ N_("You asked to amend the most recent commit, but doing so would make\n"
 static const char empty_cherry_pick_advice[] =
 N_("The previous cherry-pick is now empty, possibly due to conflict resolution.\n"
 "If you wish to commit it anyway, use:\n"
-"\n"
-"    git commit --allow-empty\n"
-"\n");
+"    git commit --allow-empty\n");
```

Comment:
It looks like here we're just removing the extra new lines around `git commit --allow-empty`.  I don't have a strong preference whether the additional new lines actually add anything in terms of readability, however such a style change of the output should probably be a separate commit with some additional justification if you really want it.

Though really I'd suggest just keeping the current style.  Changes such as these often go through a long discussion on the mailing list, so I think it would be better to keep it entirely separate for a later point, after `cherry-pick --skip` which is the main important part here actually landed.

Reply:
So the final output should look like:
```On branch master
You are currently cherry-picking commit b5063ef.

nothing to commit, working tree clean
The previous cherry-pick is now empty, possibly due to conflict resolution.
If you wish to commit it anyway, use:

    git commit --allow-empty

and then use:

    git cherry-pick --continue

to resume cherry-picking the remaining commits.
If you wish to skip this commit, use:

    git cherry-pick --skip
```

Reply:
Right, I think that's consistent with how it is now.  I don't think the additional vertical space hurts much here.